pkg/common/utils.go

```diff
@@ -76,9 +76,9 @@ func VolumeIDToKey(id string) (*meta.Key, error) {
 func KeyToVolumeID(volKey *meta.Key, project string) (string, error) {
 	switch volKey.Type() {
 	case meta.Zonal:
-		return fmt.Sprintf(volIDZonalFmt, project, volKey.Zone, volKey.Zone), nil
+		return fmt.Sprintf(volIDZonalFmt, project, volKey.Zone, volKey.Name), nil
```

Comment:
please add unit test for `KeyToVolumeID`. We should have done that when adding the function in the first place, oops